src/qml/DockPage.qml

```diff
@@ -111,53 +111,30 @@ ItemPage {
 
             Item {
                 Layout.fillHeight: true
+                Layout.fillWidth: true
```

Comment:
why

Reply:
> why

you told me to add layout.fillwidth

Reply:
in between the label and the switch, not here

Reply:
> in between the label and the switch, not here

oh